pom.xml

```diff
@@ -681,4 +681,4 @@
             <version>${swagger.ui.version}</version>
         </dependency>
     </dependencies>
-</project>
+</project>
```

Comment:
Just revert this line change and it's good!

Reply:
This is now corrected.